cli/mtv_api_tests/run.py

```
@@ -0,0 +1,261 @@
+    Args:
+        job_yaml_path: Path to the Job YAML file.
+
+    Raises:
+        typer.Exit: If the replace command fails.
+    """
+    if not job_yaml_path.exists():
+        console.print(f"[red]Error: {job_yaml_path} not found. Run 'generate' first.[/red]")
+        raise typer.Exit(code=1)
+
+    console.print("\n[bold]Deploying Namespace, Secret, and Job...[/bold]")
+    result = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)